### PR TITLE
Fix bug 1687328: Replace Tab keyboard shortcut with Ctrl + Shift + Up/Down

### PR DESCRIPTION
**Pull request by mathjazz**
That leaves the default and expected behavior of the Tab key unchanged,
which is critical for accessibility and usability.

Also included:
- Add shortcut to the Locales tab tooltip
- Fix keyboard shortcut display in the panel

Deployed to stage:
https://mozilla-pontoon-staging.herokuapp.com/projects/firefox/all-resources/

### CODE REVIEW

**Comment by mathjazz** [comment]
@julen r?
@bidaian r?

**Comment by julen** [comment]
I have added some thoughts in the original bug, and I'd hold off merging this until discussion has been clarified there.

**Comment by julen** [comment]
Am I supposed to do something else than restarting the server? FWIW this is what I get when opening the shortcuts dialog:
Shortcuts window

Aside from this, and not strictly related to this particular issue, but I noticed how reading the dialog from top to bottom, down arrows come before up arrows (i.e. 'next' before 'prev'), which feels a bit counter-intuitive.

File: frontend/src/core/editor/reducer.js
Comment (julen) [inline frontend/src/core/editor/reducer.js:199]: This change/fix seems to be unrelated to changing the keyboard shortcut.

**Comment by mathjazz** [comment]
Sadly you need to `make build` first and then restart the server to apply the localization changes.

I've changed the order of Next/Previous.

File: frontend/src/core/editor/reducer.js
Comment (mathjazz) [reply frontend/src/core/editor/reducer.js:199]: Correct, I thought it's a small enough fix to sneak it in as a ride-a-long. :)

Comment (julen) [reply frontend/src/core/editor/reducer.js:199]: My personal preference is to keep such changes aside in order to make `git blame` useful in the future. Your call nonetheless.

**Review by julen — APPROVED** [review]
Looks and works good.

As a follow-up question remains if a focused textarea should be required in order to trigger these actions or not.

**Comment by mathjazz** [comment]
> As a follow-up question remains if a focused textarea should be required in order to trigger these actions or not.

I'd leave that unchanged for now and open a separate bug if needed. I'd be open to investigate pros/cons of such change.

**Pull request merged**
